### ISSUE TITLE
as an operator I want the clusternator server to persist its state between reloads

### ISSUE THREAD

**Issue by bennett000** [rangle/the-clusternator#204]
The clusternator server is largely stateless, but there are a few small pieces of state that do need to persist across sections.  This can be done with _existing_ dynamodb code.

We can keep this simple, and store only the following:

- user
    - username
    - salt/hashed password
- project
    - github hook token
    - salt/hashed project token (for CircleCI POST requests)
    - slack channel

@rafkhan I'm going to add you to this issue b/c you already did the github table, and can most easily modify that code to work with the schema above.  we will need to meet briefly to make sure we're on the same page with this.

**Comment by bennett000** [comment]
@rafkhan lets review this tonight b/c the user section has some peculiarities.  This might really be a "3", not a "5", but I went with "5" b/c of the user peculiarities

**Comment by bennett000** [comment]
I added a new issue #221 that more completely describes the direction I think the DDB wrappers should go.  I still need to provide #204 (this issue) with info about tokens, authorities, users, and passwords.  

If #221 works like I expect, dealings with projects, tokens, authorities, users, and passwords can be done without too many code changes.  _However_ it _might_ be suboptimal in terms of DDB design, wrt hot keys etc.

Storing tokens in DDB seems silly, in the longer run I would like to ship clusternator with a paired docker redis instance.  That way we could run multiple clusternator servers in parallel if we wanted to.
